src/react-native/05-intro-to-testing/intro-to-testing.md

```diff
@@ -39,10 +43,49 @@ If you’ve implemented the solution correctly, the tests will pass when you run
 <details>
 <summary>Click to see the solution</summary>
 
+✏️ Update **src/App.test.tsx** to be:
+
+@diff ../../../exercises/react-native/05-intro-to-testing/01-solution/App.test.tsx ../../../exercises/react-native/05-intro-to-testing/01-solution/App.test.tsx only
+
+</details>
+
+## Objective 2: Write a test for handling user interactions
+
+TODO
+
+### Concept TODO
+
+TODO
+
+### Setup 2
+
+✏️ Update **src/App.tsx** to be:
+
+@diff ../../../exercises/react-native/05-intro-to-testing/01-solution/App.tsx ../../../exercises/react-native/05-intro-to-testing/02-problem/App.tsx only
+
+### Verify 2
+
+✏️ Update **src/App.test.tsx** to be:
+
+@diff ../../../exercises/react-native/05-intro-to-testing/01-solution/App.test.tsx ../../../exercises/react-native/05-intro-to-testing/02-problem/App.test.tsx only
```

Comment:
Is this diff correct? There are more lines highlighted than I would expect.

<img width="741" alt="Screenshot 2024-04-30 at 7 07 25 PM" src="https://github.com/bitovi/academy/assets/10070176/228f27a0-6a01-47f6-b849-d90e3e8d1c49">